### PR TITLE
A new hope

### LINKED ISSUE #14: Update the repository and bootstrap the connector
- [ ] README
- [ ] Update namespaces
- [ ] Setup Travis-CI

### PR DESCRIPTION
**Pull request by pamil**
Closes #14.

### CODE REVIEW

**Comment by aRn0D** [comment]
@pamil Why do we use phpunit rather than phpspec? I saw that you remove `Bundle` from the namespace, why do you plan to organize that bundle/library?

File: composer.json [hunk 1]
```diff
@@ -1,50 +1,34 @@
...
+            "email": "kamil@kokot.me",
+            "homepage": "http://kamil.kokot.me"
         }
     ],
-    "minimum-stability": "dev",
     "require": {
-        "akeneo/pim-community-dev": "~1.2",
-        "guzzlehttp/guzzle":        "~3.8"
+        "akeneo/pim-community-dev": "^1.7",
+        "symfony/symfony": "^2.7"
```
Comment (jmleroux) [inline composer.json]: `symfony/symfony` is already a PIM dependency. Do you really need to add it here?

Comment (fitn) [inline composer.json]: Yes it does. Here are the extension dependencies not the PIM ones ;)

**Comment by pamil** [comment]
@aRn0D I used PHPUnit for now as I wasn't designing the code but just testing framework-related code. There's nothing stopping us from using PHPSpec along with PHPUnit :)

As for removing `Bundle` namespace, I've just started a brand new project and haven't thought about it, if we decide it is really needed, it can be added in no time, just simple find-and-replace.

**Pull request merged**